spec/game_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'spec_helper'
+
+describe "game" do
+
+  describe "Guess!" do
+    before :each do
+      dictionary = Dictionary.new %w(apple banana cheese danish egg fry)
+      @game = Cinch::Plugins::Game.new(dictionary,dictionary, solution: "banana")
+    end
+
+    it "can tell when you guess too high" do
+      expect(@game.guess!("apple")).to eql(:missed_north)
+    end
+
+    it "can tell when you guess too low" do
+      expect(@game.guess!("cheese")).to eql(:missed_south)
+    end
+
+    it "can tell when you guess something not a word" do
```

Comment:
"not in the dictionary" -- and this test could be elaborated to include distinctions between ref dict and gameplay dict